src/dmd/declaration.d

```diff
@@ -1088,6 +1088,8 @@ extern (C++) class VarDeclaration : Declaration
     Expression edtor;               // if !=null, does the destruction of the variable
     IntRange* range;                // if !=null, the variable is known to be within the range
 
+    VarDeclarations* maybes;        // STC.maybescope variables that are assigned to this STC.maybescope variable
```

Comment:
* Can this be a D array?
* The comment should be a Ddoc comment
* Can we call this `maybeScopes`, `possibleScopeVariables` or some variations of those?

Reply:
1. No, as it's variable sized, and we generally avoid D arrays in the front end for that reason.
2. As the rest or not, such a change should await that
3. I'm not a big fan of long names

Reply:
> I'm not a big fan of long names

I'm not a fan of names that are not understandable. `maybes`, "maybe" what?

Reply:
> No, as it's variable sized, and we generally avoid D arrays in the front end for that reason.

There are several cases where D arrays are used [1] [2] and you're usually arguing for D string instead of C strings, which are D arrays.

[1] https://github.com/dlang/dmd/blob/master/src/dmd/dclass.d#L56
[2] https://github.com/dlang/dmd/blob/master/src/dmd/dclass.d#L189